### ISSUE TITLE
File does not contain a valid streaming url

### ISSUE THREAD

**Issue by vuyibiran** [y20k/transistor#5]
Hello

Thanks for making this app.

I tried it and run into problems. Transistor seems to fail to play some streams. 
At least it failed playing this one:
http://www.dradio.de/streaming/dlf_hq_ogg.m3u

Hope you can fix it.

**Comment by y20k** [comment]
Hi there,

during development I got the impression that the official Android MediaPlayer library, that I use, does not support OGG streams. That's why Transistor looks explicitly for the string "ogg" in the streaming URL embedded into m3u files and subsequently displays "File does not contain a valid streaming url".

Now that the App is more or less finished (and since you asked), I looked into the matter and it seems that at least your URL does not pose a problem for Transistor :-). I just deleted two lines checking for "ogg" and I seemed to work

I need to do some testing, but I guess I will incorporate "ogg-support" in the next version.

Best wishes.

![2016-01-11 15 28 10](https://cloud.githubusercontent.com/assets/9103935/12237984/d4d64a52-b881-11e5-999b-4b84df761304.png)
![2016-01-11 15 28 36](https://cloud.githubusercontent.com/assets/9103935/12237983/d4d2b356-b881-11e5-8559-05f55904bef1.png)
